php/libraries/Readme_NDB_Notifier.md

```diff
@@ -0,0 +1,46 @@
+## Notifier Class
```

Comment:
Markdown READMEs don't really belong in the libraries directory. This should be a doc comment in the file itself so that it's with the code (or ideally described in some higher level document for new LORIS developers explaining the code.)